models/well_construction/screen_component.py

```diff
@@ -0,0 +1,193 @@
+        ScreenMaterialTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenMaterial',
+        help_text="Material that makes up the screen. E.g. metal,"
+                  "steel, iron, copper, brass, bronze, everdur,"
+                  "Armco metal, veriperm, stone, plastic, PVC,"
+                  "ABS, Fibreglass, etc.")
+
+    screen_internal_diameter = models.ForeignKey(
+        Quantity, null=True, blank=True,
+        related_name='screen_internal_diameter',
+        on_delete=models.SET_NULL, verbose_name='screenInternalDiameter',
+        help_text="Internal screen diameter.")
+
+    screen_external_diameter = models.ForeignKey(
+        Quantity, null=True, blank=True,
+        related_name='screen_external_diameter',
+        on_delete=models.SET_NULL, verbose_name='screenExternalDiameter',
+        help_text="External screen diameter.")
+
+    screen_perforation_method = models.ForeignKey(
+        PerforationMethodTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenPerforationMethod',
+        help_text="Method used for perforating the screen. E.g."
+                  "drill, grinder, axe / chisel, machine, saw, torch,"
+                  "other, etc.")
+
+    screen_fitting = models.ForeignKey(
+        ScreenFittingTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenFitting',
+        help_text="The screen fitting (from the bottom). E.g. bail,"
+                  "open, plug, tail pipe, washdown, etc.")
+
+    screen_make = models.ForeignKey(
+        ScreenMakerTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenMake',
+        help_text="Make of the screen.")
+
+    screen_model = models.ForeignKey(
+        ScreenModelTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenModel',
+        help_text="Model of the screen.")
+
+    screen_number = models.ForeignKey(
+        ScreenNumberTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenNumber',
+        help_text="Screen number corresponds to hole size and is"
+                  "given in 0.001 inch. The value is expressed "
+                  "as an alphanumeric code.")
+
+    screen_placement = models.ForeignKey(
+        ScreenPlacementTerm, null=True, blank=True,
+        on_delete=models.SET_NULL, verbose_name='screenPlacement',
+        help_text="Screen placement method. E.g. bail down, pull"
+                  "back, jetted, washed down, etc.")
+
+    screen_wall_thickness = models.ForeignKey(
+        Quantity, null=True, blank=True,
+        related_name='screen_wall_thickness',
+        on_delete=models.SET_NULL, verbose_name='screenWallThickness',
+        help_text="Thickness of the screen wall.")
+
+    # TODO:
+    #   need to ask about this construction component is
+    #   just 1 for a screen or many?
+    construction_component = models.ForeignKey(
```

Comment:
@meomancer I think this is one to one?
<img width="1040" alt="Screen Shot 2020-07-02 at 12 57 50" src="https://user-images.githubusercontent.com/26101337/86321664-a4bf3500-bc63-11ea-8f70-044d0b65a7ee.png">
<img width="1001" alt="Screen Shot 2020-07-02 at 12 59 18" src="https://user-images.githubusercontent.com/26101337/86321764-d932f100-bc63-11ea-85bf-5a02be5ebb6e.png">